inc/storefront-template-functions.php

```diff
@@ -159,14 +159,20 @@ function storefront_header_widget_region() {
 	 * @return void
 	 */
 	function storefront_site_branding() {
-		if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
-			the_custom_logo();
+    if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
+      $logo = get_custom_logo();
+
+      echo $logo = is_home() ? '<h1>' . $logo . '</h1>' : $logo;
```

Comment:
If the h1 wrapper is preferred, let's also tweak `storefront_site_branding()` so that that `site-branding` wrapping div also wraps the logo. We can then use that class for all layout purposes. Then it should just be a case of removing the logo specific styles which is easy enough.